namerd/docs/storage.md

```diff
@@ -40,6 +40,13 @@ The Kubernetes storage plugin does not support TLS.  Instead, you should run `ku
 which will create a local proxy for securely talking to the Kubernetes cluster API. See (the k8s guide)[https://linkerd.io/doc/latest/k8s/] for more information.
 </aside>
 
+
+**Deprecation Notice**:
+As of Kubernetes 1.8+, The ThirdPartyResource API has been [deprecated](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource/) in favor of CustomResourceDefinitions
+API. Kubernetes 1.7 supports both API to help migrate ThirdPartyResource objects to CustomResourceDefinition objects. More details on how to migrate to the new API in Kubernetes 1.8+ can be found
+[here](https://kubernetes.io/docs/tasks/access-kubernetes-api/migrate-third-party-resource/). 
+
+
 **How to check ThirdPartyResource is enabled**
```

Comment:
this section below should be updated/replaced

Reply:
I added the Deprecation Notice as a way to notify people of the change so that they could migrate their DTabs if they were using K8s 1.7. I can remove this if it is no longer necessary.

Reply:
ah, the Deprecation Notice is good. i was referring to the section titled `How to check ThirdPartyResource is enabled`. if we're still supporting `ThirdPartyResource`'s for k8s <= 1.7, i suppose this gets left as is?

Reply:
We may have to make the distinction on that line, something like "for Kubernetes <= v1.7". Hopefully it shouldn't be too confusing.